.github/workflows/pr-gates.yml

```
@@ -26,7 +26,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4 # v6
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # v6
         with:
```

YAMLlint (1.38.0)

[error] duplication of key "with" in mapping

(key-duplicates)